Gemfile

```diff
@@ -6,6 +6,7 @@ gemspec
 # Specify the same dependency sources as the application Gemfile
 gem("activesupport", "~> 5.2")
 gem("railties", "~> 5.2")
+gem("vernier", "~> 0.4.0") if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("3.2.1")
```

Comment:
Usually, gems have multiple gemfiles to test against [like this](https://github.com/Shopify/identity_cache/tree/80741177e6e036424308a97ab11188885ac7aeeb/gemfiles). We can use a 2.7 gemfile and assume the root gemfile will be using the latest ruby.

Reply:
Note to self - bump this to 0.5.0 now that it has been released